src/EnergyPlus/FanCoilUnits.cc

```diff
@@ -3944,7 +4021,11 @@ namespace FanCoilUnits {
 				Node( InletNode ).MassFlowRateMinAvail = AirMassFlowLow;
 				FanFlowRatio = LowSpeedRatio;
 				Calc4PipeFanCoil( FanCoilNum, ZoneNum, FirstHVACIteration, QUnitOutMaxLS );
-				FanElecPowerLS = FanElecPower;
+				if ( FanCoil( FanCoilNum ).FanType_Num != DataHVACGlobals::FanType_SystemModelObject ) {
+					FanElecPowerLS = Fans::GetFanPower( FanCoil( FanCoilNum ).FanIndex );
+				} else {
+					FanElecPowerLS = HVACFan::fanObjs[ FanCoil( FanCoilNum ).FanIndex ]->fanPower();
```

Comment:
@Myoldmopar It is computed and it is not an input, so I think there is a strong case for keeping the private variable and the getter. If the getter is getting called a lot, then maybe it should get inlined rather than as it is now.